vortex-expr/src/expressions.rs

```diff
@@ -125,15 +125,15 @@ pub fn lit<T: Into<Scalar>>(n: T) -> Value {
 impl Value {
     // NB: We rewrite predicates to be Field-op-predicate, so these methods all must
     // use the inverse operator.
-    pub fn eq(self, field: impl Into<FieldPath>) -> Predicate {
+    pub fn equals(self, field: impl Into<FieldPath>) -> Predicate {
```

Comment:
why the rename? not_equals is so wordy

Reply:
Because of the `same_name_method` lint. I explicitly put `allow` for that one in cases where it was essentially a wrapper to match new trait signature (e.g., `len()` and `slice()`) had that in a few cases. This conflicts with `eq` from `PartialEq` though, which kind of has different semantics

Reply:
not sure I'm a fan of same_name_method but let's roll with it for now